core/src/main/scala/slamdata/SbtSlamDataBase.scala

```diff
@@ -426,12 +438,72 @@ abstract class SbtSlamDataBase extends AutoPlugin {
       },
 
       resolvers ++= {
-        if (publishAsOSSProject.value)
+        if (!publishAsOSSProject.value)
           Seq(Resolver.bintrayRepo("slamdata-inc", "maven-private"))
         else
           Seq.empty
-      }
-    )
+      },
 
+      trickleCreatePullRequest := {
+        val prior = trickleCreatePullRequest.value
+        val log = sLog.value
+
+        { (repo: OutdatedRepository) =>
+          import cats.effect.{ContextShift, IO}
+
+          import github4s.Github
+          import github4s.domain.NewPullRequestData
+
+          import java.nio.file.Files
+          import scala.concurrent.ExecutionContext.Implicits.global
+
+          implicit val cs: ContextShift[IO] = IO.contextShift(global)
+
+          prior(repo)
+
+          val authenticated = {
+            val uri = new URI(repo.url)
+            s"${uri.getScheme}://${sys.env("GITHUB_ACTOR")}:${sys.env("GITHUB_TOKEN")}@${uri.getHost}${uri.getPath}"
+          }
+
+          val dir = Files.createTempDirectory("sbt-slamdata")
+          if (runWithLogger(s"git clone --depth 1 $authenticated ${dir.toFile.getPath}", log) != 0) {
+            sys.error("git-clone exited with error")
+          }
+
+          val branchName = s"trickle/${System.currentTimeMillis}"
+          if (runWithLogger(s"cd $dir; git checkout -b $branchName", log) != 0) {
+            sys.error("git-checkout exited with error")
+          }
+
+          val managed = ManagedVersions(dir.resolve(VersionsPath))
+          repo.updates foreach {
+            case ModuleUpdateData(_, _, newRevision, repo, _) =>
+              managed(repo) = newRevision
+          }
+
+          if (runWithLogger(s"cd $dir; git commit -a -m 'Applied dependency updates' --author='SlamData Bot <bot@slamdata.com>'", log) != 0) {
+            sys.error("git-commit exited with error")
+          }
+
+          if (runWithLogger(s"cd $dir; git push origin $branchName", log) != 0) {
+            sys.error("git-push exited with error")
+          }
+
+          val createPrF = Github[IO](sys.env.get("GITHUB_TOKEN"))
+            .pullRequests
+            .createPullRequest(
+              "slamdata",
+              repo.repository,    // TODO
```

Comment:
That's not going to work with the default I set. Repository names will come out as `slamdata-tectonic`, etc.